blockchain/chain/src/store/index.rs

```diff
@@ -17,6 +17,70 @@ const DEFAULT_CHAIN_INDEX_CACHE_SIZE: NonZeroUsize =
 /// Configuration which sets the length of tipsets to skip in between each cached entry.
 const SKIP_LENGTH: ChainEpoch = 20;
 
+// this module helps speed up boot times for forest by checkpointing previously seen tipsets from snapshots.
+pub(super) mod checkpoint_tipsets {
+    use cid::Cid;
+    use forest_blocks::TipsetKeys;
+    use once_cell::sync::Lazy;
+    use std::collections::HashMap;
+    use std::str::FromStr;
+
+    macro_rules! add_calibnet {
+        ($map: ident, $key_hash:expr) => {
+            $map.insert(
+                $key_hash,
+                // calibnet genesis tipset keys
+                TipsetKeys::new(vec![Cid::from_str(
+                    "bafy2bzacecyaggy24wol5ruvs6qm73gjibs2l2iyhcqmvi7r7a4ph7zx3yqd4",
```

Comment:
Just to confirm, does this hash match the one from rebooted calibnet?

Reply:
yes, genesis keys are always same.